Core/GDCore/Project/ResourcesManager.h

```diff
@@ -402,6 +402,48 @@ class GD_CORE_API BitmapFontResource : public Resource {
   gd::String file;
 };
 
+/**
+ * \brief Describe a tilemap file used by a project.
+ *
+ * \see Resource
+ * \ingroup ResourcesManagement
+ */
+class GD_CORE_API TilemapResource : public Resource {

```

Comment:
Great idea to have this tilemap resource! Will be much cleaner 👍

Reply:
I think so too. Having one interface for both file types makes it much simpler to use